docs/index.asciidoc

```diff
@@ -1,6 +1,19 @@
 = APM Node.js Agent Reference (Experimental)
 
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/nodejs/current/index.html[elastic.co]
+endif::[]
 
-== APM Node.js Agent
-
-Welcome to the APM Node.js agent docs. The documentation pages are still work in progress. For more details also check the APM Node.js Agent https://github.com/elastic/apm-agent-nodejs[Github repository].
+include::./intro.asciidoc[]
```

Comment:
add an empty line between each include. The book might build OK right now, but if you ever want to do advanced stuff like using generated IDs, the extra line break will save you from a lot of hair as you debug an obscure issues.